### PR TITLE
Add Hello Microverse Project Files to the repo

### PR DESCRIPTION
**Pull request by IsaMarvin**
# :heavy_plus_sign: Added Following Files

- :heavy_check_mark: **a `.gitignore` file** 
- :heavy_check_mark: **`index.html` file**
- :heavy_check_mark: **`style.css` file**
- :heavy_check_mark: **`license file`**
- :heavy_check_mark: **images folder**
- :heavy_check_mark: **linter files**
- :heavy_check_mark: **Readme file**: customized with my (@IsaMarvin)  own data.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -46,17 +46,3 @@ jobs:
...
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup ESLint
-        run: |
-          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
-          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
-      - name: ESLint Report
-        run: npx eslint .
```
Comment (Whoistolu) [inline .github/workflows/linters.yml:62]: - [ ] [REQUIRED] Kindly note that you are not allowed to make custom changes to linters configuration. You are required to set up linters correctly without altering them. Linters help to ensure code standard adherence.  Kindly check [Linters config](https://github.com/microverseinc/linters-config) for more details.

File: index.html [hunk 2]
```diff
@@ -6,15 +6,15 @@
...
+  <link rel="stylesheet" href="css/style.css">
+  <title>Hello-World | A Hello World Microverse Project</title>
 </head>
 
 <body>
-  <h1>Hello, World!</h1>
+  <h1 class="heading">Hello, World!</h1>
 </body>
 
 </html>
```
Comment (Whoistolu) [inline index.html:20]: - [ ] [REQUIRED] One of the project requirements states that you should add a `.gitignore file ` and ensure that `test.md` file and `node_modules` directory are been enlisted within it. Please implement this in your project to make your work look more professional.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @IsaMarvin,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

#### To Highlight:
✔️ Descriptive PR
✔️ Meaningful commit messages
✔️  Descriptive README

_Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

#### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: .github/workflows/linters.yml (hunk 1 above)
Comment (IsaMarvin) [reply .github/workflows/linters.yml:62]: - There are no changes done to the linters file. Basically, ESlint is not required for this project so I used the linters file mentioned in the project requirements you can find it [here](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) and you can compare the two files.

Comment (Whoistolu) [reply .github/workflows/linters.yml:62]: Dear @IsaMarvin Kindly note that you are not allowed to alter the linters. You deleted the eslint code for the `linters.yml` file. Also note that the project requires you to add a `css` file alongside an `html` file. Eslint simply checks for errors on your `css` file so it should not be removed.

File: index.html (hunk 2 above)
Comment (mateo951) [reply index.html:20]: Addressed ✔

File: .github/workflows/linters.yml (hunk 1 above)
Comment (mateo951) [reply .github/workflows/linters.yml:62]: All good 📝

**Review by mateo951 — APPROVED** [review]
### Hi @IsaMarvin, 

Well done implementing all the changes required by the previous reviewer ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 😁 
### Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**